wiki/Modding/Kudosu/en.md

```diff
@@ -0,0 +1,7 @@
+# Kudosu
+
+**Kudosu** is a reward system for [Modding](/wiki/Modding). It awards points for successful modposts on beatmap discussion <!-- TODO: beatmap discussion stub -->pages. Accumulated points can be seen on a user's profile-page.
+
+In Modding v1 <!-- TODO: modding v1 stub -->, Kudosu was exchangeable for Star Priority. <!-- TODO: Star Priority stub? -->
```

Comment:
should be referred to as "forum modding" or something equivalent, "Modding v1" was never a name for it